app/Http/Resources/LeaderboardUserResource.php

```diff
@@ -19,7 +21,9 @@ class LeaderboardUserResource extends JsonResource
 {
     public function toArray($request): array {
         return [
-            'username'       => $this->user->username,
+            'id'             => (int) $this->user->id,
+            'displayName'    => (string) $this->user->name,
+            'username'       => (string) $this->user->username,
             'profilePicture' => ProfilePictureController::getUrl($this->user),
```

Comment:
Maybe we can include the LightUserResource here. This reduced the amount of different resources with similar attributes.

What do you think?

Reply:
I would prefer my proposed implementation because it fits better into my existing implementation of the user model.